scripts/release.sh

```diff
@@ -310,6 +310,49 @@ maybedryrun "${dry_run}" cat "${release_json_file}" |
 log
 log "Release workflow started successfully!"
 
+log
+log "Would you like for me to create a pull request for you to automatically bump the version numbers in the docs?"
+while [[ ! ${create_pr:-} =~ ^[YyNn]$ ]]; do
+	read -p "Create PR? (y/n) " -n 1 -r create_pr
+	log
+done
+if [[ ${create_pr} =~ ^[Yy]$ ]]; then
+	pr_branch=autoversion/${new_version}
+	title="docs: bump ${channel} version to ${new_version}"
+	body="This PR was automatically created by the release script.
+
+Please review the changes and merge if they look good and the release is complete.
+
+You can follow the release progress [here](https://github.com/coder/coder/actions/workflows/release.yaml) and view the published release [here](https://github.com/coder/coder/releases/tag/${new_version}) (once complete)."
+
+	log
+	log "Creating branch \"${pr_branch}\" and updating versions..."
+
+	create_pr_stash=0
+	if ! git diff --quiet --exit-code -- docs; then
+		maybedryrun "${dry_run}" git stash push --message "scripts/release.sh: autostash (autoversion)" -- docs
+		create_pr_stash=1
+	fi
+	maybedryrun "${dry_run}" git checkout -b "${pr_branch}" "${remote}/${branch}"
+	execrelative go run ./release autoversion --channel "${channel}" "${new_version}" --dry-run
+	maybedryrun "${dry_run}" git add docs
+	maybedryrun "${dry_run}" git commit -m "${title}"
+	# Return to previous branch.
+	maybedryrun "${dry_run}" git checkout -
+	if ((create_pr_stash)); then
+		maybedryrun "${dry_run}" git stash pop
+	fi
+
+	log "Creating pull request..."
+	maybedryrun "${dry_run}" gh pr create \
+		--assignee "@me" \
+		--reviewer bpmct,stirby \
```

Comment:
Can we define these in an arg as default values?

Reply:
Are you suggesting we define these at the top of the script? Or that we add CLI flags for them? Or perhaps neither of the aforementioned?

Reply:
Either one works, so long as the reviewers can be overridden if/when we need to without modifying the script

Reply:
I find it hard to imagine a scenario where being able to change this without modifying the script is useful 🤔. Even if it's there, it wouldn't be used and thus forgotten. The PR can always be edited on GitHub afterwards.

That's to say, I don't mind moving these vars to the top, and I can even add hidden envs for it, but not sure it's worth the effort to flag/document it.

Reply:
322135510